WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/HelpAndSupportViewController.swift

```diff
@@ -296,6 +309,19 @@ private extension HelpAndSupportViewController {
     func rowAtIndexPath(_ indexPath: IndexPath) -> Row {
         return sections[indexPath.section].rows[indexPath.row]
     }
+
+    /// Opens custom help center URL in a web view
+    ///
+    func launchCustomHelpCenterWebPage() {
+        guard let customHelpCenterURL = customHelpCenterURL else {
+            return
+        }
+
+        WebviewHelper.launch(customHelpCenterURL, with: self)
+
+        ServiceLocator.analytics.track(.supportHelpCenterViewed,
+                                       withProperties: ["help_content_url": customHelpCenterURL.absoluteString])
```

Comment:
nit: if we pass `sourceTag` to `HelpAndSupportViewController`, we can perhaps log the source tag as well in case the URL gets updated in the future.

Reply:
Good point about logging `sourceTag` instead of URL. 👍 

1. As mentioned in the comment https://github.com/woocommerce/woocommerce-ios/pull/7553#discussion_r953286459, we will be presenting custom help content for screens that will not have a support source tag. So, I thought it will be better to log the URL to be consistent. 
2. Also, I am not sure whether Android follows this `sourceTag` pattern. (As we aren't really using it for Zendesk yet) 

What do you think?

Reply:
Replying here for other similar comments - If there are sources for the Help Center other than from WPAuthenticator, how about we create a different enum that includes the cases we support in `WordPressSupportSourceTag` and also ones in the app? Then it's not really tied to the `sourceTag` pattern in iOS.

Reply:
Thanks for your inputs, @jaclync!

We decided to track the `step` and `flow` values from the Authenticator library. For screens from WCiOS, I will use custom values for `step` and `flow` as those screens won't have `Step` and `Flow`. 

I will request a review again once I make those changes. 🙏 
 
Internal ref - pe5sF9-sZ-p2#project-details